Makefile

```diff
@@ -167,4 +173,4 @@ purge: stop clean
 stop:
 	docker-compose down -v
 
-.PHONY: default build serve initdb shell tests docs deps travis-deps clean purge debug stop
+.PHONY: default build serve initdb shell tests docs deps travis-deps clean purge debug stop list
```

Comment:
Did you mean to include this line?

Reply:
Was not meant to be included, f5d7e08 fixes this.

Reply:
Looks like it got re-added in 8c8349f though.

Reply:
should be fixed now.